addons/account/models/account_tax.py

```diff
@@ -807,14 +812,20 @@ def _compute_taxes(self, base_lines, tax_lines=None,
             currency = line_vals['currency'] or self.env.company.currency_id
 
             if taxes:
-                taxes_res = taxes.compute_all(
+
+                if handle_price_include is None:
+                    manage_price_include = bool(line_vals['handle_price_include'])
+                else:
+                    manage_price_include = handle_price_include
+
+                taxes_res = taxes.with_context(**line_vals['extra_context']).compute_all(
```

Comment:
The taxes are managed price included by default on the bank reco widget. So on a line of 1000 with 15% taxes, the amount is changed to 869,57 with a tax line of 130.43. At this point, the user is free to edit the base amount as he want. This is exiting the "price included" mode and then, taxes are computed like a regular journal entry. Since you are able to mix lines with "price included" & "price excluded" mode, I need:
1) handle_price_included=False per line (for price_excluded mode)
2) force_price_included=True inside the context + handle_price_included=True (for price_included mode)